### PR TITLE
`Development`: Adjust playwright e2e tests

### PR DESCRIPTION
**Pull request by az108**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

#### Client
- [x] I **strictly** followed the [client coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] I documented the TypeScript code using JSDoc style.

### Motivation and Context
Migrated Tests from Cypress had to be adjusted in Playwright. The corresponding cypress tests were skipped.

### Description
Adds system health tests to Playwright and disables them in Cypress.

### Steps for Testing
- **Code Review**: Ensure that tests pass for valid reasons, functionality is adequately tested and check the code quality.
- **Run the tests (optional)**: Tests are checked by automatic run in CI environment. You can optionally run them locally and check if they pass.

**Steps for running the tests:**
1. Navigate to **src/tests/playwright**
2. Configure Playwright using playwright.env file based on your local setup. Current configuration should work for default Artemis setup.
3. Run `npm install && npm run playwright:setup && npx playwright test e2e/SystemHealth.spec.ts`
4. Confirm that **"System health"** tests pass

#### Code Review
- [x] Code Review 1
- [x] Code Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

## Summary by CodeRabbit

- **Tests**
	- Introduced tests for checking the health status of system components using Playwright.
	- Temporarily skipped system health check tests in Cypress.
	- Added a new test script `SystemHealth.spec.ts` for checking system health indicators on a web page.

### CODE REVIEW

File: src/test/playwright/support/pageobjects/exam/ExamManagementPage.ts [hunk 1]
```diff
@@ -101,11 +101,11 @@ export class ExamManagementPage {
         await expect(this.page.locator('#student-exam .datatable-body-row', { hasText: username }).locator('.submitted')).toHaveText('Yes');
     }
 
-    async checkQuizSubmission(courseID: number, examID: number, username: string, score: string) {
+    async checkQuizSubmission(courseID: number, examID: number, username: string, score1: string) {
```
Comment (muradium) [inline src/test/playwright/support/pageobjects/exam/ExamManagementPage.ts:104]: Out of curiosity - why `score1` instead of `score`? 😄

**Review by muradium — DISMISSED** [review]
Code LGTM, tests pass locally 👍

**Review by egekurt123 — APPROVED** [review]
Besides @muradium s comment looks good to me

**Review by sarpsahinalp — DISMISSED** [review]
Tests pass locally!

File: src/test/playwright/support/pageobjects/exam/ExamManagementPage.ts (hunk 1 above)
Comment (az108) [reply src/test/playwright/support/pageobjects/exam/ExamManagementPage.ts:104]: i forgot to revert the change ups. I first had an assertion of 2 partial strings thats why score 1 and score 2 😄

**Review by sarpsahinalp — APPROVED** [review]
Reapprove

**Comment by krusche** [comment]
two playwright tests fail. Can you check if this is connected to this PR?

**Review by Jan-Thurner — APPROVED** [review]
tests work as expected, and code lgtm.

**Review by egekurt123 — APPROVED** [review]
Reapprove

**Review by muradium — APPROVED** [review]

**Review by Jan-Thurner — APPROVED** [review]
reprove

**Review by egekurt123 — APPROVED** [review]

**Review by pzdr7 — APPROVED** [review]
Code

**Review by muradium — APPROVED** [review]
New changes fix the communication tests

**Review by sarpsahinalp — APPROVED** [review]
Reapprove

**Pull request merged**